texts/0004-ipc-router.md

```diff
@@ -0,0 +1,76 @@
+- Feature Name: `ipc_router`
```

Comment:
I wonder if this file should be removed? At first glance it looks like this is the actual RFC for the IPC router with this file (and wouldn't be merged as-is I'm assuming)